src/utils/parsers/common.ts

```diff
@@ -359,7 +367,7 @@ export abstract class BaseParser {
         const matches = this.getAllMatches(fileContent)
 
         for (const match of matches) {
-            const usage = this.extractUsageInformation(filteredFile, match)
+            const usage = this.extractUsageInformation(file, filteredFile, match)
```

Comment:
what if we parsed the content here based on the output line numbers? rather than passing both the file and the filteredFile

Reply:
or made it's own method to parse the content, given the file and the resulting line numbers